project_description/views/project_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<record id="project_form" model="ir.ui.view">
+    <field name="name">Project description</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.edit_project"/>
+    <field name="arch" type="xml">
+        <xpath expr="//page[@name='team']" position="before">
+            <page name="description" string="Description">
```

Comment:
name it `description_page` for being more accurate and descriptive when inheriting.